sdk/python/kfp/dsl/tasks_group.py

```diff
@@ -150,6 +153,16 @@ def __init__(self) -> None:
             is_root=False,
         )
 
+    def _get_oneof_id(self) -> int:
```

Comment:
nit: this seems to be an public method, so maybe no underscore prefix?

Reply:
Yes, makes sense.